src/highscore.cpp

```diff
@@ -239,10 +239,8 @@ void HighScore::render() {
 		"PRESS ESC"
 	);
 
-	if (sceneData.gameMode == sceneNS::GM_CLASSIC) {
-		renderClassicHS();
-	}
-	else {
+	if (sceneData.gameMode == sceneNS::GM_TIME_ATK)
 		renderTimeAtkHS();
-	}
+	else
+		renderClassicHS();
```

Comment:
It displays `classic` high scores when going into the high scores scene from the home menu